components/tabs/tabs.js

```diff
@@ -603,7 +598,10 @@ class Tabs extends LocalizeCoreElement(ArrowKeysMixin(RtlMixin(FocusVisiblePolyf
 				selected: panel.selected,
 				state: state
 			};
-			if (tabInfo.selected) selectedTabInfo = tabInfo;
+			if (tabInfo.selected) {
+				selectedTabInfo = tabInfo;
+				tabInfo.activeFocusable = true;
```

Comment:
I'm a bit less sure about this piece, as I'm not 100% sure how one adds a new selected panel. Do we need to clear `activeFocusable` from all of the other tabs when this happens?

Reply:
If they try to add a second selected tab then I think it's an invalid state - what happens in that case now... do we do anything at all?

Reply:
It doesn't look like it, but I didn't come up with a scenario that tested this particular scenario. I can contrive something on main to see what happens here and try to reproduce that in the new logic.